client/pdf_generator.py

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/python3
+import time
+import os
+
+from reportlab.lib.enums import TA_JUSTIFY
```

Comment:
All of these imports from reportlab should be wrapped in a try except ImportError block. If the imports fail there should be a variable `has_reportlab` that get set to false. This variable should then be in the modules metadata to show that the module is not compatible without that library. See the [blink1 plugin](https://github.com/securestate/king-phisher-plugins/blob/master/client/blink1.py#L7-L13) for an example.